From 034274a51e8544b18add7a34eeaf3bdaa74c9bcd Mon Sep 17 00:00:00 2001
From: Richard W.M. Jones <rjones@redhat.com>
Date: Mon, 25 Jun 2012 15:19:19 +0100
Subject: [PATCH 2/3] fish: Allow mount-local test to be skipped with environment variable.

The test uses FUSE, so we need a way to disable it on RHEL 5.
---
 fish/test-mount-local.sh |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/fish/test-mount-local.sh b/fish/test-mount-local.sh
index a71db04..c1b3f55 100755
--- a/fish/test-mount-local.sh
+++ b/fish/test-mount-local.sh
@@ -18,6 +18,11 @@
 
 # Test guestfish mount-local / mount-local-run commands.
 
+if [ -n "$SKIP_TEST_MOUNT_LOCAL_SH" ]; then
+    echo "$0: skipping test because SKIP_TEST_MOUNT_LOCAL_SH is set."
+    exit 0
+fi
+
 # Skip if no FUSE.
 
 test -w /dev/fuse || {
-- 
1.7.4.1

